flows/actions/send_msg.go

```diff
@@ -32,19 +32,11 @@ const TypeSendMsg string = "send_msg"
 //	  "text": "Hi @contact.name, are you ready to complete today's survey?",
 //	  "attachments": [],
 //	  "all_urns": false,
-//	  "templating": {
```

Comment:
we could just go back to the 13.3 format of a `templating` object with a UUID and a list of variables.. but I never liked having an arbitrary UUID on a thing just so it could be localized... so I'm putting `template` and `template_variables` directly on the action object.